api/disruption_types_test.go

```diff
@@ -291,3 +292,127 @@ var _ = Describe("DisruptionStatus.RemoveTargets Test", func() {
 		})
 	})
 })
+
+var _ = Describe("Disruption Annotations Tests", func() {
+	var disruption *v1beta1.Disruption
+	var owner metav1.Object
+
+	BeforeEach(func() {
+		disruption = &v1beta1.Disruption{
+			ObjectMeta: metav1.ObjectMeta{
+				Annotations: make(map[string]string),
+			},
+		}
+		owner = &metav1.ObjectMeta{
+			Annotations: map[string]string{
+				"key1": "value1",
+				"key2": "value2",
+			},
+		}
+	})
+
+	Describe("CopyOwnerAnnotations", func() {
+		It("copies annotations from the owner to the disruption", func() {
```

Comment:
nit: try to use multiple input with a `DescribeTable` to verify different success scenarios.